frontend/src/movie-gallary/movie-gallary-info/movie-gallary-info.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import './movie-gallary-info.css';
+
+const API = 'https://reactjs-cdp.herokuapp.com/movies/';
+
+export default class MovieInfoDescription extends React.Component {
```

Comment:
I don't see this component is used anywhere really and can't be tested. Dead code?